delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/projects/Project.scala

```diff
@@ -0,0 +1,36 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.model.projects
+
+import java.util.UUID
+
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Label
+import org.apache.jena.iri.IRI
+
+/**
+  * A project representation.
+  *
+  * @param label             the project label
+  * @param uuid              the project unique identifier
+  * @param organizationLabel the parent organization label
+  * @param organizationUuid  the parent organization unique identifier
+  * @param description       an optional description
+  * @param apiMappings       the API mappings
+  * @param base              the base IRI for generated resource IDs
+  * @param vocab             an optional vocabulary for resources with no context
+  */
+final case class Project(
+    label: Label,
+    uuid: UUID,
+    organizationLabel: Label,
+    organizationUuid: UUID,
+    description: Option[String],
+    apiMappings: Map[String, IRI],
+    base: IRI,
+    vocab: IRI
+) {
+
+  /**
+    * @return a project label reference containing the parent organization label
+    */
+  def projectRef: ProjectRef =
```

Comment:
isn't `def ref: ProjectRef` enough?
since most of the times the code would look like
```
val project = ...
project.ref
```

Reply:
yep